source/rfc/ansible-style-guide.rst

```diff
@@ -0,0 +1,183 @@
+Ansible Style Guide
+===================
+
+Purpose
+-------
+
+The purpose of this document is to define and describe the style guide to be used for Ansible
+in contrail-project-config and contrail-zuul-jobs projects. Topics include but are not limited
+to: role tasks separation between task files, task definition, interpolation syntax etc.
+
+This guide is inspired by `whitecloud's ansible style guide <https://github.com/whitecloud/ansible-styleguide>`_.
+
+Rules
+-----
+
+YAML File Format
+^^^^^^^^^^^^^^^^
+
+YAML files should begin with the YAML marker `---` marking the start of the document and end with a newline.
```

Comment:
Specify whether this means newline at the end of the last line or a blank last line ;)

Reply:
Done.